packages/js/core/src/types/Uri.ts

```diff
@@ -194,4 +179,16 @@ export class Uri {
       throw Error(`Unknown uri type, cannot convert. ${JSON.stringify(uri)}`);
     }
   }
+
+  // $start: Uri-toString
+  /** @returns Uri string representation */
+  public toString(): string /* $ */ {
```

Comment:
Why do we need this?

Reply:
I'll assume that this would render the URI as a string within interpolated strings, but that's as far as I can fathom this.

TBH I've only now seen this and it's something that's been in the Uri class before, it's just been moved from lines 104-115 to here.

Reply:
Yeah I noticed that but not sure we need this functions in the first place